usaspending_api/idvs/tests/data/idv_test_data.py

```diff
@@ -181,9 +181,18 @@ def create_idv_test_data():
             recipient_unique_id="duns_%s" % (7000 + award_id),
         )
 
-        mommy.make("submissions.SubmissionAttributes", submission_window=dsws, toptier_code=str(award_id).zfill(3))
+        # reporting_fiscal_year is used to delete in an Agency Slug test
```

Comment:
Can you explain what you mean by this comment?

Reply:
Nevermind, I see the test now. Personally the comment confused me more than it helped. Would probably either call out the test or just remove it all together since all of the data in our fixtures are for a specific tests.

Reply:
I went ahead and just removed the comment.